lib/tasks/issue_emails.rake

```diff
@@ -0,0 +1,19 @@
+namespace :issue_emails do
+  desc "send emails to missing mentors"
+  task missing_mentors: :environment do
+     email_template = EmailTemplate.find_by(recipients: 4)
```

Comment:
please delete this file as those email will be sent when user is reported as missing